db/sql/cassandra/config.go

```diff
@@ -119,3 +120,8 @@ func CreateSessionFromConfig(config Config) (*gocql.Session, error) {
 func HostsAsString(hostArr []string) string {
 	return strings.Join(hostArr, ",")
 }
+
+// NewGockleSession returns a new gockle Session
+func NewGockleSession(s *gocql.Session) (session gockle.Session) {
```

Comment:
We need to unify CreateSessionFromConfig with NewGockleSession
What about having:
- NewSessionFromConfig;
- NewSessionFromConfigGocql?